map/util/lanelet2_map_preprocessor/schema/transform_maps.schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Transforming Maps",
+  "type": "object",
+  "definitions": {
+    "transform_maps": {
+      "type": "object",
+      "properties": {
+        "x": {
```

Comment:
There should be schema for map paths as well:

```
        "llt_map_path": {
          "type": "string",
          "description": "Path pointing to the input lanelet2 file",
          "default": ""
        },
        "pcd_map_path": {
          "type": "string",
          "description": "Path pointing to the input point cloud file",
          "default": ""
        },
        "llt_output_path": {
          "type": "string",
          "description": "Path pointing to the output lanelet2 file",
          "default": ""
        },
        "pcd_output_path": {
          "type": "string",
          "description": "Path pointing to the output point cloud file",
          "default": ""
        },
```

Reply:
I've modified it as you said. Thank you.